test/appium/views/profile_view.py

```diff
@@ -502,21 +509,24 @@ def backup_recovery_phrase(self):
         self.ok_got_it_button.click()
         return recovery_phrase
 
-    def edit_profile_picture(self, file_name: str):
+    def edit_profile_picture(self, file_name: str = '', remove: bool = False):
```

Comment:
would be great to have add_profile_picture, and remove_profile_picture methods instead of this one

Reply:
don't you think it's redundant?